development/migrations/dependencies.rst

```diff
@@ -0,0 +1,100 @@
+============
+Dependencies
+============
+
+Dependencies tell the Migrator what order Migrations must be installed in.
+
+Declaring Dependencies
+======================
+
+In your Migration file, add a static public function named depends_on(), returning an array.
+
+.. code-block:: php
+
+    static public function depends_on()
+    {
+        return array();
+    }
+
+All you must do is add the class name of the Migration that the current Migration depends on.
```

Comment:
"All that is needed is to add the class name(s) of the Migration(s) that the current Migration depends on."